tcms/testruns/tests/test_admin.py

```diff
@@ -1,9 +1,10 @@
 # -*- coding: utf-8 -*-
 # pylint: disable=invalid-name
+from parameterized import parameterized
```

Comment:
This is not added in devel.txt so it doesn't get installed in CI and the test job fails.

Reply:
Still unresolved.